### PR TITLE
[dotnet] Format dotnet code according to default config

### PR DESCRIPTION
**Pull request by nvborisenko**
### Description
Fix small linter issues in `dotnet/src` folder.

### Motivation and Context
Keep code "clean".

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist

- [x] I have read the [contributing](https://github.com/SeleniumHQ/selenium/blob/trunk/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: dotnet/src/support/Events/FindElementEventArgs.cs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿// <copyright file="FindElementEventArgs.cs" company="WebDriver Committers">
+// <copyright file="FindElementEventArgs.cs" company="WebDriver Committers">
```
Comment (titusfortner) [inline dotnet/src/support/Events/FindElementEventArgs.cs:1]: what is different in these lines?

**Comment by titusfortner** [comment]
Looks good. Is there a linting tool for this?

File: dotnet/src/support/Events/FindElementEventArgs.cs (hunk 1 above)
Comment (nvborisenko) [reply dotnet/src/support/Events/FindElementEventArgs.cs:1]: Invisible character:

image

**Comment by nvborisenko** [comment]
https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-format

> `dotnet-format` is a code formatter for dotnet that applies style preferences to a project or solution. Preferences will be read from an *.editorconfig* file, if present, otherwise a default set of preferences will be used. At this time dotnet-format is able to format C# and Visual Basic projects with a subset of [supported .editorconfig options](https://github.com/dotnet/format/blob/main/docs/Supported-.editorconfig-options.md).

**Comment by titusfortner** [comment]
Hmm, it isn't working on my machine, does it need Windows?
Can we add a linting job to our CI like we have in other languages?

**Comment by nvborisenko** [comment]
It doesn't require Windows.

Many things, which are supposed to work, don't work on this solution out of the box...

**Comment by titusfortner** [comment]
What did you have to do to get it working?

**Pull request merged**